<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(136, 138, 133)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lb_img">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>631</width>
      <height>321</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/resources/img/offline.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_video">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Video - ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_stream">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>420</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Stream - ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_photoP">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>420</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Periodic Photo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_photoI">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>420</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Photo</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sB_num">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>420</y>
      <width>40</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lb_logo">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/resources/img/uergs-logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>712</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
